draft-ietf-quic-qlog-quic-events.md

```diff
@@ -2051,6 +2051,11 @@ error. For known frame types, the appropriate string value is used. For unknown
 frame types, the numerical value without variable-length integer encoding is
 used.
 
+The CONNECTION_CLOSE reason is a byte sequences, that may be possible to
+present as UTF-8. The `reason_text` and `reason_bytes` fields provide support
```

Comment:
That's not how I read RFC 9000. The field SHOULD be UTF-8, so the expectation is that it actually is. We're just creating an escape hatch here in case this assumption doesn't hold.

Reply:
Agreed - need better wording alignment

Reply:
If you're logging a reason sent by a peer you don't have any relationship with, there is no guarantee it is following the SHOULD. Therefore, it is only a possibility that the reason is presentable as UTF-8. I can't think of a more succint way to describe this possibility than the current text. If you have concrete suggestions for alternative text, please propose them :D

Reply:
Spoke offline with Marten and came up with a slight variant